projects/bp-gallery/src/graphql/APIConnector.tsx

```diff
@@ -606,9 +609,7 @@ export type Mutation = {
   createPicture?: Maybe<PictureEntityResponse>;
   createTimeRangeTag?: Maybe<TimeRangeTagEntityResponse>;
   createUploadFile?: Maybe<UploadFileEntityResponse>;
-  /** Create a new role */
   createUsersPermissionsRole?: Maybe<UsersPermissionsCreateRolePayload>;
-  /** Create a new user */
```

Comment:
These get automatically deleted every time you update the APIConnector with the API Generator and the code is pretty self-explanatory